### PR TITLE
update example for v7

### PR DESCRIPTION
**Pull request by kevinhughes27**
This updates the example app for the v7 namespace changes (#223 moved all the controllers into a namespace)

for review:
@nwtn @christinejordan

### CODE REVIEW

**Comment by christinejordan** [comment]
:+1:

File: example/app/controllers/authenticated_controller.rb [hunk 1]
```diff
@@ -1,5 +0,0 @@
-class AuthenticatedController < ApplicationController
```
Comment (nwtn) [inline example/app/controllers/authenticated_controller.rb]: are we not providing any of this stuff for them anymore, or did it just get moved elsewhere?

**Comment by nwtn** [comment]
other than my question, this looks :+1:

File: example/app/controllers/authenticated_controller.rb (hunk 1 above)
Comment (kevinhughes27) [inline example/app/controllers/authenticated_controller.rb]: Yeah this app is in the Engine so they can access it already. Developers just need to inherit from this controller for the most part anyways. In a real app you might copy it over if you need to add more methods but it doesn't serve a purpose in the example.

**Pull request merged**